doc/specs/vulkan/config/vulkan-macros/extension.rb

```diff
@@ -169,12 +169,12 @@ class EtextInlineMacro < CodeInlineMacroBase
 
 class PnameInlineMacro < ParamInlineMacroBase
     named :pname
-    match /pname:((\w[\w.]*)*\w+)/
+    match /pname:(\w+((\.|\-&gt;)\w+)*)/
 end
 
 class PtextInlineMacro < ParamInlineMacroBase
     named :ptext
-    match /ptext:((\w[\w.]*)*\w+)/
+    match /ptext:([\w\*]+(\[\])?((\.|\-&gt;)[\w\*]+(\[\])?)*)/
```

Comment:
> I'd like to see a bit of text added to the style guide explaining the allowed syntax here. 

Right, that can be arranged 😉 

The scripts should only need to allow "->" in places where only ".' was allowed before.

Yeah, it will unfortunatelly break generators of those who already expect "::" or "." substitution for "->"